index.html

```diff
@@ -350,66 +350,93 @@ <h3>Use Cases and Requirements</h3>
-<a>Holders</a> can share <a>verifiable presentations</a> that can be verified
-without revealing the identity of the <a>verifier</a> to the <a>issuer</a>.
+<a>Holders</a> can store <a>Verifiable Credentials</a> in any location, without
+affecting their verifiability and without the <a>Issuer</a> knowing anything about
+where they are stored or when they are accessed.
           </li>
           <li>
-<a>Holders</a> in <a>presentations</a> can either disclose the attributes or
-satisfy <a>derived predicates</a> requested by the <a>verifier</a>.
-<a>Derived predicates</a> are boolean conditions like greater than, less than,
-equal to, is in set, and so on.
+<a>Holders</a> can present <a>Verifiable Presentations</a>  to any <a>Verifier</a>
+without affecting authenticity of the claims and without revealing that action
+to the Issuer.
           </li>
           <li>
-<a>Verifiable credentials</a> and <a>verifiable presentations</a> are
-expressed in one or more standard, machine-readable data formats, which can
-also be extended with minimal coordination.
+A <a>Verifier</a> can verify <a>Verifiable Presentations</a>  from any <a>Holder</a>,
+containing proofs of claims from any Issuer.
           </li>
           <li>
-<a>Issuers</a> issue <a>verifiable credentials</a>, and <a>holders</a> store
-them. <a>Holders</a> offer <a>verifiable presentations</a>, and
-<a>verifiers</a> verify them. Issuing of <a>credentials</a>, storage of
-<a>credentials</a>, generation of <a>presentations</a>, sharing of
-<a>presentations</a>, and verification of <a>presentations</a> are
-each independent processes.
+Verification can't depend on direct interactions between <a>Issuers</a> and <a>Verifiers</a> . 
           </li>
           <li>
-<a>Verifiable credentials</a> can be revoked by the <a>issuer</a>.
+Verification can't reveal the identity of the <a>Verifier</a> to any Issuer. 
           </li>
-        </ul>
+          <li>
+The specification must provide a means for <a>Issuers</a>  to issue
+<a>Verifiable Credentials</a> that support selective disclosure, without requiring all
+conformant software to support that feature.
+          </li>
+          <li>
+<a>Issuers</a>  can issue <a>Verifiable Credentials</a> which support
+selectivedisclosure.
+          </li>
+          <li>
+If a <a>Verifiable Credential</a> supports selective disclosure, then <a>Holders</a>
+present proofs of claims without revealing the entire <a>Verifiable Credential</a>.
+          </li>
+          <li>
+<a>Issuers</a>  can issue <a>Verifiable Credentials</a> that are revocable.
+          </li>
+          <li>
+<a>Verifiable Credentials</a> and <a>Verifiable Presentations</a>  must be
+serializable in one or more interoperable machine-readable data formats.
+          </li>
+          <li>
+The data-model and serialization must be extendable with minimal coordination.
+          </li>
+          <li>
+Revocation can't reveal to the <a>Issuer</a> any identifying information about
```

Comment:
-1, as this would imply that revocation can only be done in Zero Knowledge, we don't want to limit the specification to that.

Reply:
Hmm, come to think of it, I don't understand what this statement is getting at.

Reply:
"can't" should be "SHOULD NOT" or "MUST NOT" or something non normative. Don't know what you're going for here.

Reply:
the implication is that the Issuer shouldn't necessarily glean insight to the use of a VC or the verifier community as a result of revocation.  It should be just as anonymous and privacy respecting as the original issuance....

Reply:
I agree with @stonematt here.

Reply:
"Issuer shouldn't necessarily glean insight to the use of a VC or the verifier community as a result of revocation" -- I don't understand this sentence... and I don't understand how it is related to "Revocation can't reveal to the <a>Issuer</a> any identifying information about the <a>Subject</a>, the <a>Holder</a>, the specific <a>Verifiable Credential</a>, nor the Verifier."

Are you saying, "*When a Verifiable Credential's status is checked, for example to determine if the credential has been revoked*, the process should not reveal to the <a>Issuer</a> any identifying information about the <a>Subject</a>, the <a>Holder</a>, the specific <a>Verifiable Credential</a>, nor the Verifier."

The language you're using is confusing because it makes it seem like you could be saying "the act of revocation" vs. "the act of checking to see if a credential has been revoked". I think you mean the latter, but the language you're using isn't clear about that.

Also, it depends on the status checking process and ZKP is not the only mechanism implementers are using to ensure privacy preservation. Herd privacy and blockchain-based revocation lists are two other approaches. In all 3 approaches, you can determine who the verifier is if you have control over the website/gateway/or blockchain node, for example. So, I think softening the wording is appropriate. We all know what we're going for, but the details matter and make, for example, preventing the verifier from being identified in all cases not possible.

Reply:
Perhaps "Checking revocation status shouldn't reveal to the issuer" etc.